### ISSUE TITLE
azurerm_time_series_insights_standard_environment all tiers not present

### ISSUE THREAD

**Issue by AurelienConte** [terraform-providers/terraform-provider-azurerm#8453]
Hi !

### **Community Note**
 - Please vote on this issue by adding a 👍 reaction to the original issue to help the community and maintainers prioritize this request
 - Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
 - If you are interested in working on this issue or have submitted a pull request, please leave a comment
Terraform (and AzureRM Provider) Version

### **Affected Resource(s)**
`azurerm_iot_time_series_insights_standard_environment`

### **Terraform Configuration Files**

```tf
resource "azurerm_iot_time_series_insights_standard_environment" "timeseries" {
  count = var.deploy_timeseries ? 1 : 0

  data_retention_time = "P7D"
  location = var.location
  name = "${var.environment}-${var.namespace}-timeseries"
  resource_group_name = azurerm_resource_group.resource_group.name
  sku_name = "L1_1"

  tags = {
    environment: var.environment
  }
}
```

### **Actual Behavior**

![image](https://user-images.githubusercontent.com/39769262/93094689-87b2d380-f6a2-11ea-85ab-fb1d6e21f63d.png)
(Screen of my terraform apply command)

### **Excpected Behavior**

Accept L1 sku_name, used on the portal.

![image](https://user-images.githubusercontent.com/39769262/93094827-b2049100-f6a2-11ea-8fa7-32d7ad2f810d.png)
(Screen of my Gen2 timeSeries)

Thanks in advance !

**Comment by akirakakar** [comment]
There is an issue supporting gen2, https://github.com/terraform-providers/terraform-provider-azurerm/issues/8246

**Comment by catriona-m** [comment]
Thanks for this @AurelienConte! This has now been resolved in #9616, so this can be closed.
